app/src/main/kotlin/com/simplemobiletools/calculator/helpers/BackgroundCurrencyTaskBuilder.java

```diff
@@ -0,0 +1,31 @@
+package com.simplemobiletools.calculator.helpers;
+
+import android.os.AsyncTask;
+
+import com.simpletools.calculator.commons.helpers.Calculator;
+
+public class BackgroundCurrencyTaskBuilder {
```

Comment:
why is this class in java. Any reason in specific?

Reply:
Mockito has issues with mocking Kotlin classes. To fix this problem, this class has been implemented in Java.

Reply:
Good to know. We might have to look elsewhere than mockito for the next sprints